python/ray/worker.py

```diff
@@ -1094,7 +1094,7 @@ def connect(node,
         # TODO(qwang): Rename this to `worker_id_str` or type to `WorkerID`
         worker.worker_id = _random_string()
         if setproctitle:
-            setproctitle.setproctitle("ray_worker")
+            setproctitle.setproctitle("ray_worker::IDLE")
```

Comment:
By the way I think there are some unit tests that check the proc title that you have to fix.